scater-normalize-from-counts.R

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env Rscript 
+
+# Load optparse we need to check inputs
+
+suppressPackageStartupMessages(require(optparse))
+
+# Load common functions
+
+suppressPackageStartupMessages(require(workflowscriptscommon))
+
+# parse options
+
+option_list = list(
+  make_option(
+    c("-i", "--input-object-file"),
+    action = "store",
+    default = NA,
+    type = 'character',
+    help = "File name in which a serialized R SingleCellExperiment object where object matrix found"
+  ),
+  make_option(
+    c("-e", "--exprs-values"),
+    action = "store",
+    default = NA,
```

Comment:
default should be 'counts' to reflect function default. It's a bit more complicated depending on what's available, but that's a sensible start.